app/views/components/docs/accordion.yml

```diff
@@ -0,0 +1,127 @@
+name: Accordion
+description: A group of expanding/collapsing sections
+body: |
+  The accordion has several features:
+
+  - Opening a section changes the URL so that if the URL is shared or the user uses the back button on the next page, the page will load at that part of the accordion with it expanded.
+  - If JavaScript is disabled the accordion expands fully. All of the accordion functionality (including the icons and aria attributes) are added using JavaScript.
+accessibility_criteria: |
+  The accordion must:
+
+  - indicate to users that each section can be expanded and collapsed
+  - be usable with a keyboard
+  - be fully expanded and not marked as expandable if JavaScript is disabled
```

Comment:
Maybe "JavaScript is unavailable"?

Or:
```
When javascript is unavailable the accordion must:

- be fully expanded
- not be marked as expandable
```

Reply:
I think there's some missing criteria regarding:
* open all button behaviour
* individual section expansion and collapse
* use of button rather than link (eg use of spacebar to open/close)